src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -206,13 +228,16 @@ public static LocalDate parseDate(String date) throws ParseException {
         String dateValidation = "\\d{2}-\\d{2}-\\d{4}";
```

Comment:
This is a bit arcane to the clueless. Since there is a `Date` regex in the `Appointment` class, perhaps we could move this there? We'll have to rename the regex already there since it's actually a `DateTime` regex.